core/templates/pages/exploration-player-page/learner-experience/supplemental-card.component.spec.ts

```diff
@@ -201,4 +201,24 @@ describe('Supplemental card component', () => {
     expect(componentInstance.getFeedbackAudioHighlightClass()).toEqual(
       ExplorationPlayerConstants.AUDIO_HIGHLIGHT_CSS_CLASS);
   });
+
+  it('should update displayedCard', () => {
```

Comment:
when? This has the same description as one other test in this file. PTAL

Reply:
Done, thanks for catching that.